collections/file_systems/roles/xfs_quota/readme.rst

```diff
@@ -0,0 +1,66 @@
+Xfs_quota
```

Comment:
Can you use markdown for Readme?
In BlueBanquise 2.0, I switched to markdown. 🙂